python/cudf/cudf/core/frame.py

```diff
@@ -48,6 +47,11 @@
 from cudf.utils.nvtx_annotation import _cudf_nvtx_annotate
 from cudf.utils.utils import _array_ufunc, _warn_no_dask_cudf
 
+if TYPE_CHECKING:
+    import types
```

Comment:
Let's make this `from types import ModuleType` since it's only used once. That would help avoid having such a generic name like `types`.

Reply:
Sounds good, changed